src/uwtools/config/tools.py

```diff
@@ -38,6 +37,57 @@ def compare_configs(
     return cfg_a.compare_config(cfg_b.data)
 
 
+def config_check_depths_dump(config_obj: dict, target_format: str) -> None:
+    """
+    Check that the depth of the input config does not exceed the target format's max.
+
+    :param config_obj: The reference config object.
+    :param target_format: The target format.
+    :raises: UWConfigError on excessive input-config depth.
+    """
+    cfgclass = format_to_config(target_format)
+    if cfgclass.DEPTH and depth(config_obj) > cfgclass.DEPTH:
+        msg = "Invalid depth-%s input for type-'%s' config" % (
+            depth(config_obj),
+            cfgclass.depth,
```

Comment:
Should this be `target_format` instead of `cfgclass.depth`? It's being interpolated into the text `type-'%s'`, so would we want a name instead of a number here? Similarly below...

Reply:
True - originally it was comparing depths, but this is clearer